mmv1/third_party/terraform/services/dataplex/data_source_dataplex_data_quality_rules_test.go

```diff
@@ -0,0 +1,208 @@
+package dataplex_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func TestAccDataplexDataQualityRules(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"project":       envvar.GetTestProjectFromEnv(),
+		"random_suffix": acctest.RandString(t, 10),
+		"location":      envvar.GetTestRegionFromEnv(),
+		"data_scan_id":  "tf-test-datascan-profile-id",
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataplexDataQualityRules_datascan_config(context),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("google_dataplex_datascan.tf_test_datascan_profile", "project", context["project"].(string)),
```

Comment:
I believe the checks here might be overkill, since we are just intending to test the data source. Same with the ImportState steps below.